### PR TITLE
Implement tl_classifier.py

### LINKED ISSUE #65: Use `frozen_graph.pb` inside `tl_classifier.py`
To perform inference

### PR DESCRIPTION
**Pull request by carlosgalvezp**
Using a pre-trained frozen TensorFlow model.
Currently the accuracy is not good for red lights.
Fixes #65

### CODE REVIEW

**Comment by SharkyZg** [comment]
Good job!

**Review by SharkyZg — APPROVED** [review]
